server/controllers/ssotRetrievalController.js

```diff
@@ -16,34 +16,34 @@ exports.getSingleSourceOfTruth = async (req, res) => {
     }
 };
 
-// GET /getAvailableBotsForUser/:userid
+// GET /getAvailableBotsForUser/78d09f66d2ed466cf20b06f7
```

Comment:
why was this comment to explain the schema changed?

Reply:
Erik wanted to put example calls in the comments, so he replaced the variable name with a real userId.

Reply:
We usually had real examples in those comments, so I wanted to keep that style